### PR TITLE
Thermal Dynamics & Thermal Foundation (Read commit names)

### PR DESCRIPTION
**Pull request by 016Nojr**

### CODE REVIEW

File: _data/recipes.yml [hunk 1]
```diff
@@ -1,6 +1,8 @@
 sites:
   mc-wiki:
     url: 'https://minecraft.gamepedia.com/'
+  td-docs:
```
Comment (Tonius) [inline _data/recipes.yml]: Kind of a nitpick, but please put this below `te-docs`

File: _data/recipes.yml [hunk 2]
```diff
@@ -397,6 +439,111 @@ items:
...
+  nugget-copper:
+    name: 'Copper Nugget'
+    sprite: [0, 10]
+    site: 'tf-docs'
+    url: 'metals-and-alloys/copper/'
+  nugget-diamond:
+    name: 'Diamond Nugget'
+    sprite: [1, 11]
+    site: 'tf-docs'
+    url: 'materials/nuggets'
```
Comment (Tonius) [inline _data/recipes.yml]: Add trailing slash

File: _data/recipes.yml [hunk 3]
```diff
@@ -397,6 +439,111 @@ items:
...
+  nugget-electrum:
+    name: 'Electrum Nugget'
+    sprite: [10, 10]
+    site: 'tf-docs'
+    url: 'metals-and-alloys/electrum/'
+  nugget-emerald:
+    name: 'Emerald Nugget'
+    sprite: [2, 11]
+    site: 'tf-docs'
+    url: 'materials/nuggets'
```
Comment (Tonius) [inline _data/recipes.yml]: Add trailing slash

File: _data/recipes.yml [hunk 4]
```diff
@@ -177,11 +179,51 @@ items:
...
+    name: 'Ender Pearl'
+    sprite: [12, 9]
+    site: 'mc-wiki'
+    url: 'Ender_Pearl'
+  eye-of-ender:
+    name: 'Eye of Ender'
+    sprite: [13, 9]
+    site: 'mc-wiki'
+    url: 'Eye_of_Ender'
+  filter:
```
Comment (Tonius) [inline _data/recipes.yml]: Rename this entry (not the name) to `filter-basic`, even though that's not the ingame name, it's more clear here

File: _data/recipes.yml [hunk 5]
```diff
@@ -492,6 +644,31 @@ items:
     sprite: [7, 0]
     site: 'mc-wiki'
     url: 'Redstone'
+  retriever:
```
Comment (Tonius) [inline _data/recipes.yml]: Rename this entry (not the name) to `retriever-basic`, even though that's not the ingame name, it's more clear here

File: _data/recipes.yml [hunk 6]
```diff
@@ -582,6 +759,31 @@ items:
     sprite: [6, 8]
     site: 'mc-wiki'
     url: 'Sand'
+  servo:
```
Comment (Tonius) [inline _data/recipes.yml]: Rename this entry (not the name) to `servo-basic`, even though that's not the ingame name, it's more clear here

File: _data/recipes.yml [hunk 7]
```diff
@@ -1014,6 +1216,101 @@ recipes:
       secondary-output:
         item: ['sulfur']
       secondary-chance: 30
+  filter:
```
Comment (Tonius) [inline _data/recipes.yml]: Add `-basic` like above

File: _data/recipes.yml [hunk 8]
```diff
@@ -1014,6 +1216,101 @@ recipes:
...
+    type: 'crafting'
+    results: 
+      - item: 'filter'
```
Comment (Tonius) [inline _data/recipes.yml]: Don't forget to update the `-basic` stuff in the recipes

File: _data/recipes.yml [hunk 9]
```diff
@@ -1014,6 +1216,101 @@ recipes:
       secondary-output:
         item: ['sulfur']
       secondary-chance: 30
+  filter:
+    type: 'crafting'
+    results: 
+      - item: 'filter' 
+    ingredients: 
+      - item: 'nugget-iron'
+        suffix: 'x2'
```
Comment (Tonius) [inline _data/recipes.yml]: Amounts don't need to be added in the ingredients column, or in the results column for that matter. I only sometimes add it in the results column to make the different ore processing recipes more distinguishable when used on the same page.

File: _data/recipes.yml [hunk 10]
```diff
@@ -1626,6 +1923,101 @@ recipes:
       primary-output:
         item: ['clathrate-ender']
         amount: 3
+  retriever:
```
Comment (Tonius) [inline _data/recipes.yml]: Add `-basic` like in `items`

File: _data/recipes.yml [hunk 11]
```diff
@@ -1685,6 +2077,101 @@ recipes:
       result:
         item: *rockwools
         amount: 1
+  servo:
```
Comment (Tonius) [inline _data/recipes.yml]: Add `-basic` like in `items`

File: docs/thermal-dynamics/attachments/covers/index.md [hunk 12]
```diff
@@ -0,0 +1,18 @@
+--- 
+title: Covers 
+--- 
+
+![](/assets/images/thermal-dynamics/covers.png "Covers on Redstone Energy Fluxducts")
```
Comment (Tonius) [inline docs/thermal-dynamics/attachments/covers/index.md]: Move the text on the right into the brackets (`[]`) on the left (remove the `""`)

Comment (Tonius) [inline docs/thermal-dynamics/attachments/covers/index.md]: The same goes for any other images you added

File: docs/thermal-dynamics/index.md [hunk 13]
```diff
@@ -16,5 +16,5 @@ transport items, fluids, [Redstone Flux](/docs/redstone-flux/), and even people.
 
 <div class="uk-margin-top uk-button-group">
     <a class="uk-button uk-button-large uk-button-success uk-text-bold" href="/downloads/">Download</a>
-    <a class="uk-button uk-button-large" href="https://github.com/CoFH/ThermalDynamics">GitHub</a>
+    <a class="uk-button uk-button-large" href="https://github.com/CoFH/ThermalDynamics" target="_blank">GitHub</a>
```
Comment (Tonius) [inline docs/thermal-dynamics/index.md]: This change shouldn't be here. Did you merge the latest commits into your fork?

**Comment by 016Nojr** [comment]
I have changed everything except for the item sprite-sheet, I dont have the proper program to convert properly from 16x16 to 32, the ones in the sheet at the moment are converted from 150x150 to 32x32 using Paint 3D

**Pull request merged**